### ISSUE TITLE
pagenation bug report

### ISSUE THREAD

**Issue by pythonwood** [geex-arts/django-jet#416]
I think jet is the greatful after comparing so many add advance interface.

So I catch a little bug.

![截图_2019-11-21_17-04-00](https://user-images.githubusercontent.com/5157880/69323194-3358e600-0c81-11ea-9f85-0ecff9108783.png)

**Comment by pythonwood** [comment]
Another one  IN Django 2.2.7 (but no bug in Django 2.0.2)
![截图_2019-11-21_17-11-03](https://user-images.githubusercontent.com/5157880/69323582-fa6d4100-0c81-11ea-907f-0ac26aa7fb74.png)
![截图_2019-11-21_17-11-18](https://user-images.githubusercontent.com/5157880/69323583-fb05d780-0c81-11ea-8742-de4b1e8ab182.png)

**Comment by clearinterface** [comment]
I don't think this is a bug. The pagination appears to be working for me. See screenshot below: I am currently using django 2.2.7 and Python 3.6.6

<img width="1655" alt="Screen Shot 2019-11-28 at 11 04 29 PM" src="https://user-images.githubusercontent.com/1323877/69843008-9b846a80-1233-11ea-8869-67a75140cee7.png">
<img width="1655" alt="Screen Shot 2019-11-28 at 11 04 41 PM" src="https://user-images.githubusercontent.com/1323877/69843009-9b846a80-1233-11ea-979f-563390a1d4a4.png">

**Comment by pythonwood** [comment]
@clearinterface Yes, but maybe lager page number  (4661)  is not good ?
